### PR TITLE
Use libvirt 4.9.0 packages from the official site

### PR DESCRIPTION
**Pull request by cynepco3hahue**
`virt-preview` repository does not have `libvirt-4.*` packages, but we still can not use `libvirt-5.*` packages because of our downstream process.
Currently, I will just download and install `libvirt-4.*` packages from the libvirt official site `https://libvirt.org/sources/`.

In additional I moved part of dependencies from the virt-launcher images to the libvirt image, to make possible to build it with bazel.

### CODE REVIEW

File: publish.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker tag kubevirt/alukiano:4.9.0 docker.io/kubevirt/libvirt:4.9.0
```
Comment (stu-gott) [inline publish.sh]: Is the source image correct here?

Comment (cynepco3hahue) [reply publish.sh]: Thanks, missed it

**Review by stu-gott — APPROVED** [review]

**Review by nertpinx — COMMENTED** [review]
I'm not sure about the version requirement due to the process. If you want to use this particular version, then installing from this locations is probably a safe choice. Although using something that is part of the distribution would be even safer, I believe.
Also this could render #9 useless and it could be closed IIUC.

**Comment by cynepco3hahue** [comment]
This PR does not relevant anymore, we decided to work with the latest libvirt versions.

**Pull request closed without merging**